### PR TITLE
fix assorted typos and adjust whitespace per project norms

### PR DESCRIPTION
**Pull request by garlick**
Maybe we can leave this PR open for a bit to catch typo fixes and the like for the next release.

### CODE REVIEW

File: src/modules/job-manager/plugins/perilog.c [hunk 1]
```diff
@@ -512,11 +512,11 @@ static int conf_init (flux_t *h, struct perilog_conf *conf)
                           &error,
                           "{s?{s?{s?o s?F !} s?{s?o !}}}",
                           "job-manager",
-                          "prolog",
-                            "command", &prolog,
-                            "kill-timeout", &conf->prolog_kill_timeout,
-                          "epilog",
-                            "command", &epilog) < 0) {
+                            "prolog",
```
Comment (chu11) [inline src/modules/job-manager/plugins/perilog.c:515]: Commit message typo "perlog"

Comment (garlick) [reply src/modules/job-manager/plugins/perilog.c:515]: I guess that commit was a net zero!

Comment (garlick) [reply src/modules/job-manager/plugins/perilog.c:515]: Fixed.

**Comment by garlick** [comment]
Just pushed some whitespace changes to older job manager code.

**Review by grondo — APPROVED** [review]
:+1:

**Pull request merged**